### ISSUE TITLE
what`s the meaning of the no data between the gene

### ISSUE THREAD

**Issue by lori1996** [MrOlm/inStrain#172]
<img width="960" alt="20240322092222" src="https://github.com/MrOlm/inStrain/assets/164440859/917f4938-459c-4aaa-9d54-538c2000e2b6">
Hello, I have some output files (SNV.tsv) that  don't understand. Could you please help me with it~
I see the document to explain the gene of the SNV.tsv-"If a gene file was included, this column will be present listing if the SNV is in the coding sequence of a gene". So, if this is no data, it means this SNV not in genes? but according to my outfiles, the  position 4839  of this no data SNV  is inclued in the MP9-1-complete-150_ctg297528_56x_c_3 gene. Why the position is in this gene, this SNP is detected in non-gene?

**Comment by MrOlm** [comment]
Hi @lori1996 - two possible explanations. 1) are you running an old version of inStrain? That could explain it. 2) Is it a "cryptic" SNV? If so, that could explain it.

Best,
Matt

**Comment by lori1996** [comment]
@MrOlm thank your reply. I checked the version of my software, it`s v1.8.0.  and I`m sorry that I don`t understand the mean of cryptic SNV, how can I know  the SNV is cryptic or how can I detect cryptic SNV?

**Comment by lori1996** [comment]
@MrOlm thank your reply. I checked the version of my software, it`s v1.8.0.  and I`m sorry that I don`t understand the mean of cryptic SNV, how can I know  the SNV is cryptic or how can I detect cryptic SNV?


> Hi @lori1996 - two possible explanations. 1) are you running an old version of inStrain? That could explain it. 2) Is it a "cryptic" SNV? If so, that could explain it.
> 
> Best, Matt

@MrOlm thank your reply. I checked the version of my software, it`s v1.8.0.  and I`m sorry that I don`t understand the mean of cryptic SNV, how can I know  the SNV is cryptic or how can I detect cryptic SNV?
